lib/defaultTheme.js

```diff
@@ -5,6 +5,7 @@ const theme = {
   colors: {
     ...DefaultTheme.colors,
     primary: '#2165E3',
+    grey: '#6D6D68',
```

Comment:
as mentioned before, I believe this is the text color, already existing in the theme